spyder/config/main.py

```diff
@@ -270,6 +270,14 @@
               'show_hscrollbar': True,
               'visible_if_project_open': True
               }),
+            ('vcs',

```

Comment:
It would be useful to be able to modify these settings in the UI; is there anywhere this can be done?

Reply:
good idea. I will look into that.

Reply:
Does this happen when you pass ``--safe-mode`` to ``bootstrap.py`` (i.e. a clean preferences file)? I cannot reproduce the issue opening and closing preferences and moving through every tab on both a dirty and a clean config.

Reply:
okay, it was just due to my setup

Reply:
Thanks.